src/controllers/commands/CommandController.cpp

```diff
@@ -643,7 +643,85 @@ void CommandController::initialize(Settings &, Paths &paths)
             getApp()->windows->getMainWindow().getNotebook().getSelectedPage());
 
         currentPage->getSelectedSplit()->getChannelView().clearMessages();
+        return "";
+    });
 
+    this->registerCommand("/settitle", [](const QStringList &words,
+                                          ChannelPtr channel) {
+        if (words.size() < 2)
+        {
+            channel->addMessage(
+                makeSystemMessage("Usage: /settitle STREAM TITLE."));
+            return "";
+        }
+        if (auto twitchChannel = dynamic_cast<TwitchChannel *>(channel.get()))
+        {
+            auto status = twitchChannel->accessStreamStatus();
+            getHelix()->updateChannel(
+                twitchChannel->roomId(), "", "", words.mid(1).join(" "),
+                [channel](NetworkResult dank) {
+                    channel->addMessage(makeSystemMessage(
+                        QString("Updated title.").arg(dank.status())));
```

Comment:
unused argument

Reply:
Replaced with useful argument in 516b17ae